### ISSUE TITLE
Add implementation for remote-config

### ISSUE THREAD

**Issue by corrado4eyes** [GitLiveApp/firebase-kotlin-sdk#134]
| Library               | Class                            | Platforms                  |
| :-----------------| :------------------------ | :---------------------- |
| remote-config | RemoteConfig | Android, iOS, js  |

Create implementation for a shared version of Firebase Remote Config.

**Comment by ellsworthrw** [comment]
remote config also has a js library

**Comment by corrado4eyes** [comment]
Hi @ellsworthrw, I closed it because of my mistake the base branch was not the `GitLiveApp/master`, but instead [this one](https://github.com/splendo/firebase-kotlin-sdk/tree/splendo-fit-1.4).  Also, tests are missing. Now we're merging what we did in the past in our fork in this repository's master and then I'll make the PR for remote-config along with tests.

In the meanwhile, if you want to have a look at what I did [here](https://github.com/splendo/firebase-kotlin-sdk/tree/implementation/remote-config) is the branch where remote-config is implemented.
